pages/groups/create.js

```diff
@@ -181,7 +186,15 @@ class CreateGroup extends Component {
                 placeholder="Tell your members what you'll be playing, or maybe a bit about your GM style."
                 value={description} />
 
-              <small>Tell your members what you'll be playing, or maybe a bit about your GM style.</small>
+              <small>
```

Comment:
Maybe we should componentize this to make adding the editor help modal easier. It can track it's own state to display the dialog instead of mucking up the page.

Reply:
Yup, check #211. :-)

Reply:
Approved :)